serial_coms.py

```diff
@@ -110,8 +110,9 @@ def testLine(self, testChar):
         
 if __name__ == "__main__":
     # Tests serial by bouncing a random number through the arduino
+    microController = serialLine(38400) # Sets up serial communication line to STM32 (9600 baud)
+    microController.sendDataBytes(100, 100, 300, 100, 100, 100)
     serialTestLine = serialLine(57600)
-
     serialTestLine.testLine('z')
```

Comment:
Change `serialTestLine` to `microController` as to use the one and only `serialLine` object